Framework/Algorithms/src/CopyDataRange.cpp

```diff
@@ -17,23 +18,54 @@ using namespace Mantid::HistogramData;
 
 namespace {
 
+std::size_t getBinIndexOfValue(NumericAxis const *axis, double value) {
```

Comment:
Would it be possible to use [MatrixWorkspace::binIndexOf](https://github.com/mantidproject/mantid/blob/master/Framework/API/inc/MantidAPI/MatrixWorkspace.h#L147) directly instead? I think this is the same thing?

Reply:
`binIndexOf` appears to be doing something different - it doesn't return the right index when I tried using it!